packages/components/organisms/f-cookie-banner/static/CHANGELOG.md

```diff
@@ -3,6 +3,10 @@
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+** DEPRECATED - Please use f-cookie-banner CHANGELOG.md **
```

Comment:
Is this still deprecated if it now lives inside f-cookie-banner?

Reply:
I wasn't really sure what to do with this changelog. I didn't really want to delete it in case we needed to refer to it in the future. Thoughts?

Reply:
Hmm, good point. Let's keep it for now, we can always move it or delete it later.